launchable/commands/record/session.py

```diff
@@ -35,8 +35,15 @@
     cls=KeyValueType,
     multiple=True,
 )
+@click.option(
+    "--observation",
+    "is_observation",
+    help='observation',
+    is_flag=True,
+    required=False,
```

Comment:
[nits]
Since this is a flag that specifying `--observation` means to use observation mode, I think it is redundant to say `required=False`.